src/Admin/AbstractAdmin.php

```diff
@@ -2235,32 +2124,32 @@ public function isGranted($name, $object = null)
         return $this->cacheIsGranted[$key];
     }
 
-    public function getUrlSafeIdentifier($entity)
+    public function getUrlSafeIdentifier($entity): ?string
     {
         return $this->getModelManager()->getUrlSafeIdentifier($entity);
     }
 
-    public function getNormalizedIdentifier($entity)
+    public function getNormalizedIdentifier($entity): ?string
     {
         return $this->getModelManager()->getNormalizedIdentifier($entity);
     }
 
-    public function id($entity)
+    public function id($entity): ?string
```

Comment:
```suggestion
    public function id(object $entity): ?string
```

Reply:
This argument is passed to `getNormalizedIdentifier()` method at model manager, which seems to allow `null`.